### ISSUE TITLE
Spike: UPE.- Testing different themes

### ISSUE THREAD

**Issue by bborman22** [Automattic/woocommerce-payments#1725]
The goal of this spike is to get a broader sense of what issues the automated styles for UPE might encounter from popular WooCommerce themes. The first spike tested out 5 different free themes just to confirm that the approach would apply across different themes. This led us to determine that this approach should work, but also pointed out some gaps in it as well. This spike would be to target popular WooCommerce themes to see if any additional gaps are noticed as well give us the ability to solve some of the gaps ahead of time for as much of the market as we can.

The two steps for this spike would be in determining which themes in particular to test, perhaps focusing on our own themes, but mostly looking to target the most popular themes. Then applying those themes while using the draft PR for UPE styling and getting screenshots to notice any issues or gaps in the automated solution.

The [draft PR for styling UPE](https://github.com/Automattic/woocommerce-payments/pull/1703) is a good starting point to minimize any set up time.

**Comment by harriswong** [comment]
I tested a few popular themes from `wp-admin/theme-install.php?browse=popular`, including Astra, Elementor, OceanWP, Popular FX, Kdaence.

I also tested a few from https://woocommerce.com/product-category/themes/, such as Smart Home, Skinny, pure Fashion, Overline, Threads.

Generally, the checkout page for these themes are either a 2 columns layout (similar to Storefront), or 1 column single page layout. I confirmed all of these themes do pick up the style for the UPE element. I am also able to choose different payment methods.

I haven't test anything outside of our themes market place.
